client/landing/stepper/declarative-flow/internals/steps-repository/plans/plans-wrapper.tsx

```diff
@@ -126,6 +140,8 @@ const PlansWrapper: React.FC< Props > = ( props ) => {
 					isInVerticalScrollingPlansExperiment={ isInVerticalScrollingPlansExperiment }
 					shouldShowPlansFeatureComparison={ isDesktop } // Show feature comparison layout in signup flow and desktop resolutions
 					isReskinned={ isReskinned }
+					hidePlansFeatureComparison={ hidePlansFeatureComparison }
+					plansToShow={ plansToShow }
```

Comment:
This doesn't work since it doesn't take the terms, i.e. monthly or annually, into account, which is what the `getPlansFromProps()` function should be doing in `PlansFesaturesMain`, btw. However, it is a bit broken right now.

I'm fixing it now in https://github.com/Automattic/wp-calypso/pull/75949 . Once that's done, you'll be able to just write `planTypes={ [ TYPE_FREE, TYPE_PERSONAL, TYPE_PREMIUM] }` here. I'll circle back when it's done.

Reply:
Thank you @southp, I need some architectural help on the Plans side of things ✌️